### PR TITLE
Laravel 10 Compatibility!

### PR DESCRIPTION
**Pull request by pedro-santiago**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -20,14 +20,14 @@
   "require": {
     "php": "^8.0",
     "guzzlehttp/guzzle": "^7.4",
-    "laravel/framework": "^7.0|^8.0|^9.0"
+    "laravel/framework": "^8.0|^9.0|^10.0"
```
Comment (rafaellaurindo) [inline composer.json]: Can we maintain Laravel 7 compatibility?

**Review by rafaellaurindo — CHANGES_REQUESTED** [review]
Thanks for your contribution!

File: composer.json (hunk 1 above)
Comment (pedro-santiago) [reply composer.json]:
```suggestion
    "laravel/framework": "^7.0|^8.0|^9.0|^10.0"
```

File: composer.json [hunk 2]
```diff
@@ -20,14 +20,14 @@
...
     "php": "^8.0",
     "guzzlehttp/guzzle": "^7.4",
-    "laravel/framework": "^7.0|^8.0|^9.0"
+    "laravel/framework": "^7.0|^8.0|^9.0|^10.0"
   },
   "require-dev": {
     "brianium/paratest": "^6.3",
     "friendsofphp/php-cs-fixer": "^3.2",
-    "nunomaduro/collision": "^5.10",
+    "nunomaduro/collision": "^6.0|^7.0",
```
Comment (rafaellaurindo) [inline composer.json]:
```suggestion
    "nunomaduro/collision": "^5.10|^6.0|^7.0",
```

**Review by rafaellaurindo — APPROVED** [review]

**Pull request merged**